### PR TITLE
Notice: Undefined index: column

### PR DESCRIPTION
**Pull request by itsazzad**
This happens when any core module is replaced

### Fixed Issues (if relevant)

1. magento/magento2#<issue_number>: Issue title

### Manual testing scenarios (*)

1. ...
2. ...

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Review by ihor-sviziev — CHANGES_REQUESTED** [review]
Hi @itsazzad,
Your changes looks good to me. 
Could you add steps to reproduce of this issue and cover your changes with unit test?

File: lib/internal/Magento/Framework/Setup/Declaration/Schema/Declaration/SchemaBuilder.php [hunk 1]
```diff
@@ -193,16 +193,17 @@ private function processColumns(array $tableData, string $resource, Table $table
     {
         $columns = [];
 
-        foreach ($tableData['column'] as $columnData) {
-            if ($this->isDisabled($columnData)) {
-                continue;
-            }
+        if (array_key_exists('column', $tableData)) {
```
Comment (lbajsarowicz) [inline lib/internal/Magento/Framework/Setup/Declaration/Schema/Declaration/SchemaBuilder.php:196]: You should use `isset` instead - as it's significantly faster.

**Review by lbajsarowicz — APPROVED** [review]
Just minor performance request.
Could you cover your changes with Unit Tests and fix static code analysis?

**Comment by magento-engcom-team** [comment]
Hi @lbajsarowicz, thank you for the review.
[ENGCOM-6779](https://jira.corp.magento.com/browse/ENGCOM-6779) has been created to process this Pull Request
:eight_spoked_asterisk: @lbajsarowicz, could you please add one of the following labels to the Pull Request?

| Label | Description |
|--------------------------------|-----------------------------------------------------------------|
| ```Auto-Tests: Covered``` | All changes in Pull Request is covered by auto-tests |
| ```Auto-Tests: Not Covered``` | Changes in Pull Request requires coverage by auto-tests |
| ```Auto-Tests: Not Required``` | Changes in Pull Request does not require coverage by auto-tests |

**Comment by magento-engcom-team** [comment]
@itsazzad thank you for contributing. Please accept Community Contributors team invitation [here](https://github.com/orgs/magento/invitation) to gain extended permissions for this repository.

**Review by lbajsarowicz — CHANGES_REQUESTED** [review]
Fix static tests

**Comment by ihor-sviziev** [comment]
Hi @itsazzad,
Will you be able to update your PR?

**Comment by itsazzad** [comment]
@ihor-sviziev I will try soon.

**Comment by ihor-sviziev** [comment]
Hi @itsazzad,
Unfortunately I need to close your PR due to inactivity.
Please reopen and update if you wish to continue.
Thank you for the collaboration!

**Pull request closed without merging**